Kitodo-IdentifierGenerator/src/test/java/org/kitodo/identifiergenerator/DNBURNIdentifierGeneratorTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.identifiergenerator;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for PersistentIdentifier.
+ */
+public class DNBURNIdentifierGeneratorTest {
+
+    @Test
+    public void shouldGenerateUnifiedResourceName() throws Exception {
```

Comment:
I see problem for other libraries.
SLUB adds to numbers 'id' but for e.g. UB Mannheim not (https://digi.bib.uni-mannheim.de/seitenansicht_sammlung/dlf/874/1/ : urn:nbn:de:bsz:180-digad-2807).